### PR TITLE
[ui] feature/peg-progress

### PR DESCRIPTION
**Pull request by 59023g**
Display peg progress as a notification with a link to the blockexplorer to more clearly assure the user their peg has been submitted. 

image

This is a stopgap and in memory shared tx store or localstorage persistence tbd after launch

### CODE REVIEW

**Review by ryardley — APPROVED** [review]

**Comment by ryardley** [comment]
@59023g @iverc this what you had in mind? Can look at localstorage later probably as an effect on the store as where the cache is stored is not really an interaction layer concern.

File: ui/app/src/components/Notifications.vue [hunk 1]
```diff
@@ -7,13 +7,12 @@ export default defineComponent({
...
   setup() {
-    const { store, actions } = useCore();
-    // @ts-ignore ??
+    const { store } = useCore();
     const notifications = computed(() => store.notifications);
-
     return {
       notifications,
       removeItem(index: any) {
+        console.log(index, store.notifications);
```
Comment (iverc) [inline ui/app/src/components/Notifications.vue:15]: remove log if noe needed

**Review by iverc — DISMISSED** [review]
looks good!

**Review by ryardley — APPROVED** [review]

**Comment by 59023g** [comment]
Nice ! (I can't approve bc I opened PR)

**Review by ryardley — APPROVED** [review]

**Pull request merged**